<ui version="4.0" >
 <class>TrainerDialog</class>
 <widget class="QDialog" name="TrainerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Trainer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="margin" >
    <number>3</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QFrame" name="frmQuestion" >
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="margin" >
         <number>3</number>
        </property>
        <item>
         <widget class="QProgressBar" name="mark" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>4</number>
          </property>
          <property name="value" >
           <number>2</number>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="format" >
           <string>Level: %v</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="laQuestion" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="text" >
           <string>translation to find source word for</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frmAnswer" >
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="margin" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="laAnswer" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Box</enum>
          </property>
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frmKeypad" >
       <property name="frameShape" >
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="margin" >
         <number>3</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="1" column="0" >
           <widget class="QLabel" name="la4" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>jkl</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="la7" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>stu</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="laLeftCmd" >
            <property name="font" >
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>Cancel</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLabel" name="la5" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>mno</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="la2" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>def</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="la6" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>pqr</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLabel" name="la8" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>vw</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QLabel" name="la0" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>รครถ</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2" >
           <widget class="QLabel" name="laRightCmd" >
            <property name="font" >
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>Help</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="la9" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>xyz</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="la3" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>ghi</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="la1" >
            <property name="font" >
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text" >
             <string>. ,</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar" >
          <property name="font" >
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="maximum" >
           <number>3</number>
          </property>
          <property name="value" >
           <number>2</number>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
          <property name="format" >
           <string>Tries left: %v</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>showHints()</slot>
 </slots>
</ui>
